### DISCUSSION TITLE
LDAP group mapping cannot be configured in admin area

### DISCUSSION THREAD

**Discussion by cfelbertp** [requarks/wiki#6149] [Error / Bug Report]
With [2.5.296](https://github.com/requarks/wiki/releases/tag/v2.5.296) LDAP group mapping was introduced. As far as I understand it, there should now be additional fields for the configuration in the admin area. However, I cannot find any in the LDAP / Active Directory section.

I use Wiki.js 2.5.296 with Cloudron 7.3.6.

**Comment by q-wertz** [comment]
See my first point in https://github.com/requarks/wiki/discussions/6139

It looks like you need to add a new Authentication Strategy
